content/extractor.js

```diff
@@ -16,36 +16,122 @@
 
 // Given DOM element returns array of possible id-links to resolve.
 
+import { flatten } from 'lodash';
+
 export const TAGS_BLACKLIST = [ 'script' ];
 
-export function extractPossibleMatches ($dom) {
-  const tag = $dom.tagName.toLowerCase();
+const EMAIL_PATTERN = /([^\s@]+@[^\s@]+\.[a-z]+)/i;
```

Comment:
Well the issue is that it would parse `my email (ngotchac@gmail.com)` and match `(...)`, which is no good. Is there a case where the extension is not `[a-z]+` ?